tests/e2e/fixtures/statsFixtures.js

```diff
@@ -94,6 +94,13 @@ const seasonOneMatches = [
     startTime: dayjs('2022-01-20', 'yyyy-mm-dd').valueOf(),
     endTime: dayjs('2022-01-20', 'yyyy-mm-dd').valueOf(),
   },
+  {
+    player1: 'player1',
+    player2: 'player3',
+    winner: 'player1',
+    startTime: dayjs('2022-01-20', 'yyyy-mm-dd').valueOf(),
+    endTime: dayjs('2022-01-20', 'yyyy-mm-dd').valueOf(),
+  },
```

Comment:
Can you move this up above with the other player 1 week 1 matches?

Reply:
Done